File: src/main/java/com/github/highright1234/glacier/protocol/handler/MinecraftCipherEncoder.java
package com.github.highright1234.glacier.protocol.handler;

import io.netty.buffer.ByteBuf;
import io.netty.channel.ChannelHandlerContext;
import io.netty.handler.codec.MessageToByteEncoder;

public class MinecraftCipherEncoder extends MessageToByteEncoder<ByteBuf> {
    @Override
    protected void encode(ChannelHandlerContext ctx, ByteBuf msg, ByteBuf out) throws Exception {

    }
}

File: src/main/kotlin/com/github/highright1234/glacier/protocol/handler/MinecraftCipherEncoder.kt
package com.github.highright1234.glacier.protocol.handler

import kotlin.Throws
import java.lang.Exception
import io.netty.buffer.ByteBuf
import io.netty.channel.ChannelHandlerContext
import io.netty.handler.codec.MessageToByteEncoder

class MinecraftCipherEncoder : MessageToByteEncoder<ByteBuf?>() {
    @Throws(Exception::class)
    override fun encode(ctx: ChannelHandlerContext, msg: ByteBuf?, out: ByteBuf) {
    }
}